app/common/src/queryClient.ts

```diff
@@ -114,7 +115,8 @@ export function createQueryClient<TStorageValue = string>(
           (queryKey) => !invalidatesToAwait.includes(queryKey),
         )
 
-        for (const queryKey of invalidatesToIgnore) {
+        for (const queryKeyRaw of invalidatesToIgnore) {
+          const queryKey = queryKeyRaw.map(toValue)
```

Comment:
If these may be getters, filtering them by value above doesn't look right.

Reply:
> filtering them by value above doesn't look right

hmm... should we perhaps be slicing arrays off (and/or removing object keys) that are refs/getters?

> I think the (current) filtering is wrong without a deep comparison

IIRC, tanstack query compares query keys by some kind of stable `JSON.stringify`-adjacent representation, so they are implicitly deeply compared